x/ecocredit/spec/03_messages.md

```diff
@@ -80,3 +80,51 @@ Cancel removes a number of credits from the holder's account and also deducts th
 - `credits` must not be empty
 - `batch_denom` must be a valid batch denomination
 - `amount` must be positive
+
+## MsgSell
+
+`Sell` creates one or more sell orders. Sell orders can be created in batches 
+
++++ https://github.com/regen-network/regen-ledger/blob/a70af9b4d5da6cf065eb23232ba8020769249688/proto/regen/ecocredit/v1alpha1/tx.proto#L292-L324
+
+### Validation:
+
+- `owner` must ba a valid address, and their signature must be present in the transaction
+- `batch_denom` must be a valid credit batch denom
+- `quantity` must be a positive decimal
+- `ask_price` must be a positive integer
+
+## MsgUpdateSellOrders
+
+`UpdateSellOrders` updates one or more sell orders...
```

Comment:
why using "..."?

Reply:
I was probably planning to write more and forgot. Updated. :+1: